File: Compitibility SwiftUI in Objective C Project/SwiftUIObjectiveC/SwiftUIObjectiveC/ViewController.m
//
//  ViewController.m
//  SwiftUIObjectiveC
//
//  Created by Synesis Sqa on 28/4/25.
//

#import "ViewController.h"
#import "SwiftUIObjectiveC-Swift.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
}
- (IBAction)buttonTapped:(UIButton *)sender {
    // Swift Extension will be called
    [self displaySwiftUIWapper];
}


@end

File: ConvayTestSDK/ConvayTestSDK/ViewController.swift
//
//  ViewController.swift
//  ConvayTestSDK
//
//  Created by Raiyan Sharif on 4/1/26.
//

import UIKit
import JitsiMeetSDK

class ViewController: UIViewController {
    

    override func viewDidLoad() {
        super.viewDidLoad()
        
        let jitsiMeetViewController = JitsiMeetView()
        
//        present(jitsiMeetViewController, animated: true)
        // Do any additional setup after loading the view.
    }


    @IBAction func startMeetingTapped(_ sender: UIButton) {
            
    }
    
    @IBAction func joinMeetingTapped(_ sender: UIButton) {
        let vc = MeetingViewController()
        present(vc, animated: true)
    }
    
    
}

